client/src/holdings/AddTradeDialog.tsx

```diff
@@ -39,6 +84,35 @@ export function AddTradeDialog({
     setOpen(false);
   }
 
+  async function onSubmit(data: IFormInput) {
+    let fxRate: number = Number(data.fxRate ?? 0);
+    if (!fxRate) {
+      // An FX rate must always be provided even if the trade was made
+      // in the user's currency.
+      fxRate = holding.currency === 'GBX' ? 100 : 1;
+    }
+    const createTradeData: CreateTradeData = {
+      holding: holding.id,
+      date: new Date().toISOString().split('.')[0].replace('T', ' '),
```

Comment:
This just sets the value to be the current datetime. Needs to be the actual datetime from data.